### PR TITLE
Humanise() localization

### LINKED ISSUE #2067: Humanise() localization
## What happens now
Awkward phrasing with some localized versions of the Humanise() function, particularly for "and a half" number descriptions.

## What I'd like to happen 
More translator control over grammar within the Humanise() function.

## How it can happen
Use more complete strings for localization of Humanise() function, e.g.
```cs
String s = String.Format("nearly {0} and a half million", someInteger);
```

## EDDI Version
3.7.2-b2

### PR DESCRIPTION
**Pull request by Tkael**
Resolves #2067

### CODE REVIEW

File: Tests/SpeechUnitTests.cs [hunk 1]
```diff
@@ -279,7 +279,7 @@ public void TestSectorTranslations()
         [TestMethod]
         public void TestSpeechHumanize1()
         {
-            Assert.AreEqual("well over minus 12 thousand", Translations.Humanize(-12345));
+            Assert.AreEqual("well over negative 12 thousand", Translations.Humanize(-12345));
```
Comment (richardbuckle) [inline Tests/SpeechUnitTests.cs]: Maybe this is a UK/US thing, but over here we would say "minus", at least in everyday speech.

**Review by richardbuckle — APPROVED** [review]
Looks good code-wise. Note sure about saying "negative" instead of "minus" but I don't think it's worth adding an `en-GB` localisation for.

It might be a good idea to give the translators a heads-up on CrowdIn.

**Comment by richardbuckle** [comment]
On reflection I can't think of many cases where EDDI has to speak negative numbers besides temperatures.

File: Tests/SpeechUnitTests.cs (hunk 1 above)
Comment (Tkael) [reply Tests/SpeechUnitTests.cs]: I was thinking of negative as the opposite of positive, but upon reflection and after reviewing norms for describing temperatures both in the US and UK I have to agree. I've reverted from "negative" to "minus". o7

**Pull request merged**